Vizzuality/trase
# Fix/ranking widget

## Pivotal Tracker

https://www.pivotaltracker.com/story/show/171355113

## Description

- Added an ellipsis for texts with more than 3 lines
- Added a limit to the page width to display the data view in 1 column so we don't have problems with the widget's width.

![image](https://user-images.githubusercontent.com/9701591/75455294-98eff900-5979-11ea-9c24-b7e42fc7ebf0.png)

## Testing instructions

The example context is Brazil + Beef. Check the data view using the flow/data toggle.
